src/data/AppConfiguration.ts

```diff
@@ -7,5 +7,6 @@ export const configuration = {
       import.meta.env.VITE_FEATURE_USE_CUSTOM_VERSION_UPDATE === "true",
     demoMode: import.meta.env.VITE_FEATURE_DEMO_MODE === "true",
     betaLanguages: import.meta.env.VITE_FEATURE_BETA_LANGUAGES === "true",
+    notifications: import.meta.env.VITE_FEATURE_NOTIFICATIONS === "true",
```

Comment:
I think it's better to enable/disable notification based on cureent platform
So in configuration you set notifications.android = true
In the code you can write a wrapper what will check the current app context and will return true/false to youin this csse you don't need to edit this field manually